.github/workflows/ci.yaml

```diff
@@ -237,9 +237,9 @@ jobs:
           LOWERCASE_REPO_OWNER=$(echo "${GITHUB_REPOSITORY_OWNER}" | awk '{print tolower($0)}')
           echo "LOWERCASE_REPO_OWNER=$(echo "${GITHUB_REPOSITORY_OWNER}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
 
-          echo "OPERATOR_VERSION=${COMMIT_SHORT_SHA}-ubi8" >> $GITHUB_ENV
+          echo "OPERATOR_VERSION=${COMMIT_SHORT_SHA}-ubi9" >> $GITHUB_ENV
```

Comment:
Note: Since we only provide one flavour of image we could just drop the `-ubi9` suffix.

Reply:
Agreed. I am assuming there is a rationale behind the ubi8 suffix? If it's safe enough to drop it, I am all for the change

Reply:
@cdesionitis @shivamerla thoughts on this?

Reply:
@cdesiniotis @shivamerla thoughts on this?

Reply:
yeah, safe to drop `ubi` suffix as we don't reference those tags in the Helm charts or OLM.

Reply:
@elezar Looks like I'll have to make changes in multiple places to drop the DIST suffix. I'd like to make that change in a follow-up PR. 

I'd like to limit the changeset to ubi9 upgrade in this PR